modules/c++/math.linear/include/math/linear/MatrixMxN.h

```diff
@@ -39,7 +39,17 @@ template<typename _T> bool equals(const _T& e1, const _T& e2)
 
 template<typename _T> inline bool equals(const _T& e1, const _T& e2, _T eps)
 {
+#ifdef _MSC_VER // Visual Studio
```

Comment:
The boost code is about the same as one of the solutions from SO.  I tried it out (and kinda liked it--even if it is rather heavy on C++ templates), but it seemed like an awful lot of (isolated) code to add for this one (rather spurious) warning.

Besides, this will go away (too) if we add /wd4127 to the command-line as you're suggesting above.